File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/servers/selection/ISelectedLibraryIdentifierProvider.java
package com.lasthopesoftware.bluewater.client.servers.selection;

import com.namehillsoftware.handoff.promises.Promise;

/**
 * Created by david on 2/12/17.
 */

public interface ISelectedLibraryIdentifierProvider {
	Promise<Integer> getSelectedLibraryId();
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/library/access/session/ISelectedLibraryIdentifierProvider.kt
package com.lasthopesoftware.bluewater.client.browsing.library.access.session

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId

/**
 * Created by david on 2/12/17.
 */
interface ISelectedLibraryIdentifierProvider {
	val selectedLibraryId: LibraryId?
}
